oak/server/rust/oak_runtime/src/tests.rs

```diff
@@ -26,7 +26,7 @@ fn test_app_config() {
     );
     let got = format!("{:?}", cfg);
```

Comment:
Does prost have a way of generating text format protobufs?  (ie. could we replace this with `let got = cfg.to_text()` and leave all the asserts as-is?)

Reply:
Yeah relying on debug strings does not seem ideal. In fact, why even test via a textual representation at all? It seems the assert could compare against an actual expected struct?

Reply:
Changed tests to structs.